src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1577,9 +1554,9 @@ namespace cryptonote
   {
     TRY_ENTRY();
 
-    // load json & DNS checkpoints every 10min/hour respectively,
+    // load json & every 10min/hour respectively,
```

Comment:
"load json every 10 min."

Reply:
fixed

Reply:
"hour respectively" still doesn't make sense now that you removed DNS checkpoints.

Reply:
I'll address this in the next PR